packages/widgets/src/widgets/StakeModuleWidget/components/PositionSummary.tsx

```
@@ -358,22 +358,22 @@ export const PositionSummary = () => {
           hasPositions &&
           existingSelectedVoteDelegate?.toLowerCase() !== selectedDelegate.toLowerCase() ? (
             [
-              loadingExistingDelegateOwner || !existingDelegateOwner ? (
+              loadingExistingDelegateOwner ? (
                 <Skeleton className="w-30 h-5" />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)